docs/plugins/tailor.rst

```diff
@@ -24,17 +24,23 @@ Usage
     tailor [status]
     tailor now
     tailor materials <material> [<material> ...]
+    tailor confiscate [true|false]
 
 By default, ``tailor`` will prefer using materials in this order::
 
     silk cloth yarn leather
 
-but you can use the ``tailor materials`` command to restrict which materials
-are used, and in what order. ``tailor`` supports adamantine cloth (using the
-keyword ``adamantine``) but does not use it by default, as most players find
-adamantine too precious to routinely make into cloth. ``tailor`` does not
-support modded "cloth" types which utilize custom reactions to making clothing
-out of those cloth types.
+but you can use the ``tailor materials`` command to restrict which materials are
+used, and in what order. By default, ``tailor`` will "confiscate" (i.e., remove
+ownership and mark for dumping) equipped tattered clothing. This can be changed
```

Comment:
I thought it only confiscated equipped tattered clothing when there are replacements available.